daprdocs/content/en/developing-applications/local-development/multi-app-dapr-run/multi-app-overview.md

```diff
@@ -82,13 +85,86 @@ Logs for application and `daprd` are captured in separate files. These log files
+
+If the `createService` field is set to `true` in the `dapr.yaml` template for an app, then the `service.yaml` file is generated in the `.dapr/deploy` folder of the app.
+
+Otherwise, only the `deployment.yaml` file is generated for each app that has the `containerImage` field set.
+
+The files `service.yaml` and `deployment.yaml` are used to deploy the applications in `default` namespace in Kubernetes. This feature is specifically targeted only for running multiple apps in a dev/test environment in Kubernetes.
+
+You can name the template file with any preferred name other than the default. For example:
+
+```bash
+dapr run -k -f ./<your-preferred-file-name>.yaml
+```
+
+The following example includes some of the template properties you can customize for your applications. In the example, you can simultaneously launch 2 applications with app IDs of `nodeapp` and `pythonapp`.
+
+```yaml
+version: 1
+common:
+apps:
+  - appID: nodeapp
+    appDirPath: ./nodeapp/
+    appPort: 3000
+    containerImage: ghcr.io/dapr/samples/hello-k8s-node:latest
+    createService: true
+    env:
+      APP_PORT: 3000
+  - appID: pythonapp
+    appDirPath: ./pythonapp/
+    containerImage: ghcr.io/dapr/samples/hello-k8s-python:latest
+```
+
+> **Note:** 
+> - If the `containerImage` field is not specified, `dapr run -k -f` produces an error. 
+> - The `createService` field defines a basic service in Kubernetes (ClusterIP or LoadBalancer) that targets the `--app-port` specified in the template. If `createService` isn't specified, the application is not accessible from outside the cluster.
+
+For a more in-depth example and explanation of the template properties, see [Multi-app template]({{< ref multi-app-template.md >}}).
+
+## Logs
+
+The run template provides two log destination fields for each application and its associated daprd process:
+
+1. `appLogDestination` : This field configures the log destination for the application. The possible values are `console`, `file` and `fileAndConsole`. The default value is `fileAndConsole` where application logs are written to both console and to a file by default.
+
+2. `daprdLogDestination` : This field configures the log destination for the `daprd` process. The possible values are `console`, `file` and `fileAndConsole`. The default value is `file` where the `daprd` logs are written to a file by default.
+
+### Log file format
+
+Logs for application and `daprd` are captured in separate files. These log files are created automatically under `.dapr/logs` directory under each application directory (`appDirPath` in the template). These log file names follow the pattern seen below:
+
+- `<appID>_app_<timestamp>.log` (file name format for `app` log)
+- `<appID>_daprd_<timestamp>.log` (file name format for `daprd` log)
+
+Even if you've decided to rename your resources folder to something other than `.dapr`, the log files are written only to the `.dapr/logs` folder (created in the application directory).
+
```

Comment:
Can we add a link to the hello-Kubernetes tutorial https://github.com/dapr/quickstarts/tree/master/tutorials/hello-kubernetes for an example here and make this discoverable?